inc/vcf/summary_report_writer.hpp

```diff
@@ -110,23 +110,31 @@ namespace ebi
     class SummaryReportWriter : public ReportWriter
```

Comment:
The documentation is no longer correct: it doesn't write to stdout but to a file.

I am not sure using this class right now is the safest choice, because it is a bit inconsistent in the way it displays messages:

* Warnings are only displayed once
* Meta errors are only displayed once
* The rest of errors are always displayed

Maybe we should convert the `StdoutReportWriter` into a `FileReportWriter` and change the way warnings vs errors are handled later on, once we introduce the real summary errors with counts?

Reply:
in fact only the NoMetaDefinitionError are written only once in SummaryReportWriter, but I agree with everything else.